posthog/session_recordings/queries/sub_queries/base_query.py

```diff
@@ -25,9 +24,8 @@ def ttl_days(self):
     def property_operand(self):
         return PropertyOperatorType.AND if self._query.operand == "AND" else PropertyOperatorType.OR
 
-    @property
-    def ast_operand(self) -> type[Union[ast.And, ast.Or]]:
```

Comment:
this worked but the typing was off and was upsetting my IDE
so i switched it to be a method that wrapped the provided expr instead of asking for the operand and wrapping at the caller